src/search/components/SearchResultItem.tsx

```diff
@@ -30,17 +30,19 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
 			// tags={[{ label: 'Redactiekeuze', id: 'redactiekeuze' }, { label: 'Partner', id: 'partner' }]}
 			viewCount={412}
 			bookmarkCount={85}
-			description={result.dcterms_abstract}
+			description={result.dcterms_abstract || ''}
```

Comment:
Eventueel de type nog aanpassen voor `dcterms_abstract` dat dit undefined of null kan zijn?

Reply:
https://github.com/viaacode/avo2-typings/pull/39